docs/microscope-installation-guide.md

```diff
@@ -27,11 +27,11 @@ should launch napari (may take 15 seconds on a fresh installation) with the recO
  
 ## Install and configure `Micromanager`
 
-Install `Micromanager 2.0` nightly build `20210713` (https://micro-manager.org/Micro-Manager_Nightly_Builds). 
+Install `Micromanager 2.0` nightly build `20220901` (https://micro-manager.org/Micro-Manager_Nightly_Builds). 
 
-**Note:** We have tested recOrder with `20210713`, but most features will work with newer builds. We recommend testing a minimal installation with `20210713` before testing with a different nightly build or additional device drivers. 
+**Note:** We have tested recOrder with `20220901`, but most features will work with newer builds. We recommend testing a minimal installation with `20220901` before testing with a different nightly build or additional device drivers. 
 
-Before launching `Micromanager`, download the Meadowlark device adapters and calibration files from the [release page](https://github.com/mehta-lab/recOrder/releases/) or [directly for 0.1.0](https://github.com/mehta-lab/recOrder/releases/download/0.1.0/recOrder-0.1.0-device-drivers.zip) and place these three unzipped files into your `Micromanager` folder (likely `C:\Program Files\Micro-Manager-2.0.1_2021_07_13` or similar). 
+Before launching `Micromanager`, download the Meadowlark device adapters and calibration files from the [release page](https://github.com/mehta-lab/recOrder/releases/) and place these three unzipped files into your `Micromanager` folder (likely `C:\Program Files\Micro-Manager-2.0.1_2022_09_` or similar). 
```

Comment:
I think we can replace the MM folder name with "`C:\Program Files\Micro-Manager` or similar" to keep the instructions simple and a bit easier to maintain

Reply:
Good call. Fixed.